### ISSUE TITLE
[CAIA A11Y Research - PILOT]: Ask VA - Desktop Pilot - Form (August 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#89027]
> [!IMPORTANT]
> - **TICKET STATUS:** `Completed`
> - **PILOT DATE:** 8/1/2024 at 12 p.m. ET - [Zoom link to join](https://us06web.zoom.us/j/88162411415?pwd=GvxE2bwgYi21ppfePch8Zlkj15qZrv.1)
> - **TYPES OF ASSISTIVE TECH:** Desktop `Device Type & AT Preferred`
>     - Info on AT being tested in user sessions needed, to prep and conduct pilot.
> - **PROTOTYPE:** [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
> - **PILOT PREP NOTES:** [Pilot 1 Raw Notes - 8/1/2024](https://docs.google.com/document/d/1VIlyVQktnBBF0nzIU5db5W-nRWtR1M8mdg6l5YEmegM/edit?usp=sharing)

# CAIA A11y Support
This is a child ticket of the research ticket [87480](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87480) - created to track work, completed from this **DESKTOP** pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @coforma-jamie  (Pilot)
- @sara-amanda (Copilot)
- @rileyorr (Observer 1)
- @EvanAtCoforma (Observer 2)

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Ask VA
- OCTO-DE product owner: Becky Phung
- Product manager: CeeCee O'Connor and Terri Hollar
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Eddie Otero, Hemesh Patel
- Product/team Slack channel: #ask-va-public

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** 8/1/2024 from 12- 1 p.m. ET
- **Prototype - Figma**: [Ask VA - Figma Prototype](https://www.figma.com/proto/nI0nYMdLamogjKq9t5atj8/Ask-VA-Prototype-R4-(Staging)?node-id=2001-81942&t=qOlK188ntSKlJfwi-1)
-  **Prototype - Staging:** [Ask VA - Staging Link](https://staging.va.gov/contact-us/ask-va-too/.) - `To be used in testing, per conversation guide.`
- **[Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md)**
- **[Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md)**
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

**12 Veterans:**
- minimum of 3 sessions with **screen reader** users
- minimum of 2 sessions with **alternative navigation** users
- minimum of 1 session with a **screen magnification** user

### Recruitment Criteria

<details><summary>Toggle to view recruitment criteria</summary>

- "We'd like to recruit people who use a mix of different screen readers, such as, [NVDA, JAWS, VoiceOver (Apple) and Talkback (Android)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenreaders)." 
    - **Source:** [Cohort 1: Screen Reader Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-1-screen-reader-users)
- "We'd like to recruit people who use a mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation)." 
    - **Source:** [Cohort 2: Alternative Navigation Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-2-alternative-navigation-users)
- "We'd like to recruit people who use a mix of different screen magnification software, such as, [ZoomText, ZoomText Fusion, Magnifier (Windows) and Zoom (Apple)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenmagnification)." 
    - **Source:** [Cohort 3: Screen Magnification Users, Recruitment Criteria Ask VA, Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#cohort-3-screen-magnification-users)

</details> 

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Jamie
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Conversation%20guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Form/Research%20plan.md#recruitment-criteria)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot - _**CAIA and the team will determine what AT will be used in this session, specifically.**_
- [x] **Review prototype** in `Staging` - `Provided via convo guide`: [Staging Link](https://staging.va.gov/contact-us/ask-va-too/.)
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. [Pilot Notes Doc](https://docs.google.com/document/d/1VIlyVQktnBBF0nzIU5db5W-nRWtR1M8mdg6l5YEmegM/edit?usp=sharing)
```

**Comment by coforma-jamie** [comment]
@tygindraux what tech + device combo would you like me to use? Your options are:
- MacOS + VoiceOver + Safari
- Windows + JAWS + Edge
- Windows + NVDA + Firefox

**Comment by tygindraux** [comment]
Sorry, Jamie. I just missed this. Either Windows options would be great, if possible!
